src/components/toast/ToastContainer.tsx

```diff
@@ -0,0 +1,8 @@
+import { StyledToastContainer } from "./Toast.styles.ts";
+import "react-toastify/dist/ReactToastify.min.css";
+
+const ToastContainer = () => {
+  return <StyledToastContainer newestOnTop={false} />;
```

Comment:
this component does nothing, you can used the styled version directly in App.tsx